### PR TITLE
Update the ResizeObserver IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25160)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/348381213)

File: interfaces/ResizeObserver.idl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[Constructor(ResizeObserverCallback callback),
+    Exposed=Window]
+interface ResizeObserver {
+    void observe(Element target);
+    void unobserve(Element target);
+    void disconnect();
+};
+
+callback ResizeObserverCallback = void (sequence<ResizeObserverEntry> entries, ResizeObserver observer)
```
Comment (foolip) [inline interfaces/ResizeObserver.idl]: Sent https://github.com/WICG/ResizeObserver/pull/56 to fix a missing semicolon here, will add it manually too.

File: interfaces/ResizeObserver.idl [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[Constructor(Element target)
+]
+interface ResizeObserverEntry {
+    readonly attribute Element target;
+    readonly attribute DOMRectReadOnly contentRect;
+};
+
+[Constructor(Element target)
+]
+interface ResizeObservation {
```
Comment (foolip) [inline interfaces/ResizeObserver.idl]: There's a problem here, found existing bug: https://github.com/WICG/ResizeObserver/issues/46

**Review by foolip — APPROVED** [review]
LGTM % ResizeObservation. Maybe manually exclude it, or hold off? Either WFM.

File: interfaces/ResizeObserver.idl (hunk 1 above)
Comment (lukebjerring) [reply interfaces/ResizeObserver.idl]: Added manually for now.

**Comment by lukebjerring** [comment]
As discussed offline, the incorrect source IDL shouldn't be tweaked as part of a semi-automated import flow, because the edge-case handling or manual work will keep repeating. We should just let the tests fail, and have bad test results as the motivation to correct the source.

**Review by foolip — CHANGES_REQUESTED** [review]
Oops, needs a test to use the IDL

**Review by foolip — APPROVED** [review]
Thanks!
